src/codeExecutionWorker.js

```diff
@@ -0,0 +1,67 @@
+/* global eval */
+'use strict'
+import { connectIframe, PluginClient } from '@remixproject/plugin'
+import ethers from 'ethers' // eslint-disable-line
+import Web3 from 'web3' // eslint-disable-line
+import swarmgw_fn from 'swarmgw' // eslint-disable-line
+window.swarmgw = swarmgw_fn() // eslint-disable-line
+window.ethers = ethers
```

Comment:
I don't understand why you need to set value at the window level. Who/what can/should access this window object (it's in an hidden Iframe) ?